src/style.css

```diff
@@ -1,3 +1,81 @@
-body {
+:root {
+  --light-blue: #00acdc;
+}
+
+* {
   margin: 0;
+  font-family: Roboto, serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, 'Roboto, Arial, sans-serif'.